hack/run-local.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# run-local.sh - run/cleanup the operator with OLM
+#
+# USAGE
+#    run-local.sh -a run/cleanup
+# OPTIONS
+#    -a     Action     run/cleanup the operator installation
+#?
+
+# container tool to use with operator-sdk
+CONTAINER_TOOL=podman
+
+function stop-exec() {
+    echo "Error: $*" >&2
+    exit 1
+}
+
+# Options
+while getopts "a:" opt; do
+    case "$opt" in
+	a) action="$OPTARG" ;;
+	?) stop-exec "Unknown option"
+    esac
+done
+
+if [[ ! "$action" =~ ^run|cleanup$ ]]; then
+    stop-exec "-a Action must be \"run\" or \"cleanup\""
+fi
+
+if [ -z "$AWS_SHARED_CREDENTIALS_FILE" ]; then
+    stop-exec "env AWS_SHARED_CREDENTIALS_FILE not found"
+fi
+
+if [ -z "$KUBE_SSH_KEY_PATH" ]; then
+    stop-exec "env KUBE_SSH_KEY_PATH not found"
+fi
+
+if [ -z "$CONTAINER_REPO" ]; then
+    stop-exec "env CONTAINER_REPO not found"
+fi
+
+WMCO_ROOT=$(dirname "${BASH_SOURCE}")/..
+source $WMCO_ROOT/hack/common.sh
+
+cd $WMCO_ROOT
+OSDK=$(get_operator_sdk)
+
+# Builds the container image and pushes it to repository
+# Uses this built image to run the operator on the cluster
+# containers are tagged by branch name and it is user's responsibility to clean old/unused containers in
+# container repository as well as local system.
+case "$action" in
+    run)
+
+  TAG=$(git symbolic-ref --short HEAD)
+  OPERATOR_IMAGE=$CONTAINER_REPO:$TAG
+
+  $OSDK build "$OPERATOR_IMAGE" --image-builder $CONTAINER_TOOL
```

Comment:
I am assuming you will add the parameters for ignoring the local cache here.

Reply:
or pass an argument to expose it to the end user